test/reline/yamatanooroti/test_rendering.rb

```diff
@@ -1902,13 +1902,14 @@ def test_stop_continue
       rubyfile.close
       start_terminal(40, 50, ['bash'])
       write "ruby -I#{@pwd}/lib -rreline #{rubyfile.path}\n"
+      assert_screen(/^>/)
       write "abc\ndef\nhi"
       pid = pidfile.tap(&:rewind).read.to_i
       Process.kill(:STOP, pid) unless pid.zero?
       write "fg\n"
```

Comment:
It seems to wait to ruby waking up after ```fg```.
How about ```assert_screen(/fg\n.*>/m)```

Reply:
Looks great, thanks! added